src/worker-nodes.yml

```diff
@@ -0,0 +1,5 @@
+kind: Cluster
```

Comment:
Should we move this under `test/__test__/data/` ?

Reply:
done